src/test/java/stepDefinitions/LoginPageStepDefintionSita.java

```diff
@@ -0,0 +1,25 @@
+package stepDefinitions;
+
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+public class LoginPageStepDefintionSita {
```

Comment:
Why we have to create two step definition files ?
we can user one stepDefinition for one feature file 

for example if we have login.feature file , then we should have one step definition for this one